client/client-multi/client-c/src/platform/mem.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * mem.h
```

Comment:
All header names under platform directory must have names starting with "ext_" to make it clear that they are external to Kaa platform SDK.